fdbserver/VersionedBTree.actor.cpp

```diff
@@ -4963,8 +5177,8 @@ class VersionedBTree {
 		// While there are multiple child pages for this version we must write new tree levels.
 		while (records.size() > 1) {
 			self->m_pHeader->height = ++height;
-			Standalone<VectorRef<RedwoodRecordRef>> newRecords =
-			    wait(writePages(self, &dbBegin, &dbEnd, records, height, version, BTreePageIDRef()));
+			Standalone<VectorRef<RedwoodRecordRef>> newRecords = wait(writePages(
+			    self, PagerEventReasons::metaData, &dbBegin, &dbEnd, records, height, version, BTreePageIDRef()));
```

Comment:
```suggestion
			    self, &dbBegin, &dbEnd, records, height, version, BTreePageIDRef()));
```

`writePages()` is always writing BTree pages at level `height`.